### PR TITLE
podman-desktop: 1.13.2 -> 1.15.0

### PR DESCRIPTION
**Pull request by kachick**
Diff: https://github.com/podman-desktop/podman-desktop/compare/v1.13.2...v1.15.0

Update postPatch with https://github.com/podman-desktop/podman-desktop/commit/8af3761b41f051933726b187617d14227ab0f0b8

This PR aims to fix [failure in nixpkgs-update](https://nixpkgs-update-logs.nix-community.org/podman-desktop/2024-11-26.log)

```
patching file extensions/podman/packages/extension/package.json
substituteStream() in derivation podman-desktop-1.14.1: ERROR: pattern process.resourcesPath doesn't match anything in file 'packages/main/src/tray-animate-icon.ts'
```

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by booxter** [comment]
Thank you for the PR; thanks for including helpful details about particular changes in the commit message.

The code change look good, and I was able to build both for linux and darwin. The darwin version starts and seems to work fine. My only concern is that the commit message lines are not as wrapped as it could be (I don't see a lot of commits in the repo that have such long lines). But I'm new to nixpkgs maintenance, maybe it's acceptable here. Let me know what you think. Other than that, this is good to go.

File: pkgs/applications/virtualization/podman-desktop/default.nix [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
-    sha256 = "sha256-07lf9jy22JUT+Vc5y9Tu1nkWaXU5RTdu3GibcvQsSs8=";
+    sha256 = "sha256-pewLcukPTXijEP11i2ah8F17N3Rav8UyHrpCu9fUScg=";
   };
 
-  pnpmDeps = pnpm.fetchDeps {
-    inherit (finalAttrs) pname version src;
-    hash = "sha256-LPsNRd1c/cQeyBn3LZKnKeAsZ981sOkLYTnXIZL82LA=";
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit (finalAttrs) pname version src patches;
+    hash = "sha256-EzkAcd84JEBq6eDkh9XCGfCyeo5r2vpZSIvb2hqpsBU=";
```
Comment (kachick) [inline pkgs/applications/virtualization/podman-desktop/default.nix:29]: This hash always will be different in nixpkgs-review in x86_64-linux report.

In nixpkgs-review

```
error: hash mismatch in fixed-output derivation '/nix/store/hm84x737h5dnd3g2ky8k5hg6fkmd8ly6-podman-desktop-pnpm-deps.drv':
         specified: sha256-EzkAcd84JEBq6eDkh9XCGfCyeo5r2vpZSIvb2hqpsBU=
            got:    sha256-2wmMuymv8C1ZVeGnWf2R3t9cf1jqiFIw0KJ/m3/cxZs=
error: 1 dependencies of derivation '/nix/store/wgrqfbf8wqkhvk7br091vw2nkfpn94dg-podman-desktop-1.15.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/fiw3n0zkxpwv2zcsc6zc0rya43rs315l-review-shell.drv' failed to build
```

It succeeds in darwin. I have no idea what is this :thinking:

**Comment by kachick** [comment]
@booxter Thank you for your reviewing. :pray: 

I don't have confidence that wrapping long URLs and logs will be better or not in the commit message. So keeping now. But I can modify commit messages if required. Any suggestions welcome.

However running nixpkgs-review always fails for the x86_64-linux and `pnpm_9.fetchDeps` step with hash mismatched. I don't know how to fix this behaviors. :bow:

**Comment by kachick** [comment]
I have no idea how to fix the hash mismatch, at least this year. Sorry for closing this PR 🙇.

File: pkgs/applications/virtualization/podman-desktop/default.nix (hunk 1 above)
Comment (kachick) [reply pkgs/applications/virtualization/podman-desktop/default.nix:33]: Might be related to https://github.com/NixOS/nixpkgs/pull/350063?

**Pull request closed without merging**